apis/placement/v1beta1/stageupdate_types.go

```diff
@@ -427,7 +427,7 @@ const (
 	// Its condition status can be one of the following:
 	// - "True": The staged update run is making progress.
 	// - "False": The staged update run is waiting/paused/abandoned.
-	// - "Unknown" means it is unknown.
+	// - "Unknown": The staged update run is in the process of stopping.
```

Comment:
We have the `Reason` field to tell it's in the process of stopping. Here I'd prefer to say it's in a transitioning state.